package/kedro_viz/api/responses.py

```diff
@@ -177,11 +186,7 @@ class NamedEntityAPIResponse(BaseAPIResponse):
 
 
 class GraphAPIResponse(BaseAPIResponse):
-    nodes: List[
-        Union[
-            TaskNodeAPIResponse, DataNodeAPIResponse, ParametersNodeMetadataAPIResponse
-        ]
-    ]
+    nodes: List[NodeAPIResponse]

```

Comment:
Why was `ParametersNodeMetadataAPIResponse` removed here?

Reply:
It was added there by mistake. It was never reached as part of the `Union`